packages/www/hooks/use-api.tsx

```diff
@@ -520,7 +520,7 @@ const makeContext = (state: ApiState, setState) => {
 
     async getStreams(userId): Promise<Array<Stream>> {
       const [res, streams] = await context.fetch(
-        `/stream?userId=${userId}&streamsonly=1`
+        `/stream?streamsonly=1`
```

Comment:
I think this should still have `userId` — I'm an admin but I still want my "streams" page to only show mine.

Reply:
Sure! Missed that one 😢